tests/scripts/all.sh

```diff
@@ -2114,6 +2115,7 @@ component_test_valgrind_constant_flow () {
     scripts/config.py full
     scripts/config.py set MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND
     scripts/config.py unset MBEDTLS_USE_PSA_CRYPTO
+    scripts/config.py --force set MBEDTLS_VALGRIND
```

Comment:
Rather than annotate test cases with an unrelated test platform, I would prefer to extend the list of test suites that we skip in Valgrind components, and put the relevant test cases in separate `.data` files.

If that really doesn't make sense, we might want to create a more generic category of “slow” tests that we want to run only to check good-case functional correctness, and we're confident about memory management and constant-timeness because we run similar test cases with smaller data. That would cover things like large-key RSA, Brainpool ECC and high-iterations PBKDF2.

Reply:
I think it makes more sense to just tag the small number of tests that we want to skip. In general people work with, e.g., "RSA tests", and it makes sense for them to have them all in one file, rather separate out tests by "tests that are sometimes skipped by Valgrind". That's not a distinction people usually need to think about.

Moving into separate files also makes it harder to see that we are skipping, e.g. the large-key RSA tests, but still running some tests for short keys.